### PR TITLE
Updated zarf config

### PR DESCRIPTION
**Pull request by ldgriswold**
## Testing TODO:
- [x] certs for new domain (will self-signed certs work? using https://github.com/ldgriswold/certs)
- [x] truststore update for new domain? (keycloak needs self-signed CA for realm truststore / tls truststore?)
- [x] aws ec2 test
- [x] remove upstream_security.csv prior to closing PR

## Changes:
* Organized zarf.example.config variables by zarf package in order of README
* Added `upstream_security.csv` which lists potential security risks in upstream packages
* README.md update for metallb uds repo

### CODE REVIEW

File: upstream_security.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+repo,file,line,description,value
```
Comment (RothAndrew) [inline upstream_security.csv:1]: remove file from pr

File: zarf-config.example.yaml [hunk 2]
```diff
@@ -9,25 +9,43 @@ package:
...
+      DUBBD_VALUES: test/dubbd-values.yaml      
+
       # This tells Keycloak to use its internal database instead of an external one.
       # If you'd like to use an external one, set this to "false" and set the other KEYCLOAK_* variables appropriately.
       # See https://github.com/defenseunicorns/uds-idam#zarf-variables-configuration
       KEYCLOAK_DEV_DB_ENABLED: "true"
 
       # The domain to use. Will get used as the base for all deployments. For example, if DOMAIN is 'bigbang.dev' then keycloak will be listening at 'https://keycloak.bigbang.dev'
-      DOMAIN: bigbang.dev
+      DOMAIN: dextest.dev
```
Comment (RothAndrew) [inline zarf-config.example.yaml:32]: change back to bigbang.dev

File: zarf-config.example.yaml [hunk 3]
```diff
@@ -9,25 +9,43 @@ package:
...
+      # NEVER USE BIGBANG.DEV WITH LIVE DATA. ITS TLS SECRET KEY HAS BEEN PASSED AROUND MORE THAN A HOLIDAY FRUITCAKE THAT NOBODY WANTS.
       CERT_FILE: tls.cert
       KEY_FILE: tls.key
       KEYCLOAK_CERT_FILE: tls.cert
       KEYCLOAK_KEY_FILE: tls.key
 
-      # Used for the test pipeline
-      DUBBD_VALUES: test/dubbd-values.yaml
+      #admin credential setup for keycloak
+      KEYCLOAK_DB_USERNAME: kcadmin
```
Comment (RothAndrew) [inline zarf-config.example.yaml:44]: Set same values as the upstream packages set as their defaults

**Review by RothAndrew — DISMISSED** [review]
couple things, then LGTM

File: zarf-config.example.yaml (hunk 3 above)
Comment (ldgriswold) [reply zarf-config.example.yaml:47]: Done, with the exception of GRAFANA and NEUVECTOR, which are generated by DUBBD.  Testing in `feature/custom-domain-testing` branch.

File: zarf-config.example.yaml [hunk 4]
```diff
@@ -9,10 +9,20 @@ package:
       # The IP addresses we want to assign to each ingress gateway. You'll need to either update the /etc/hosts file
       # or ensure that your internal DNS properly maps the domain names you are using to these IP addresses.
       # Use whichever addresses make sense for your environment, they don't have to stay with these values.
+      #Ssee https://github.com/defenseunicorns/uds-capability-metallb
```
[the remark this answers is not in the document]
Comment (ldgriswold) [reply zarf-config.example.yaml]: Fixed in another commit.

**Comment by RothAndrew** [comment]
pre-commit checks are failing. Try running `pre-commit run -a` locally or add a comment to the PR with the content `/update all`

**Pull request merged**